GETTING_STARTED.md

```diff
@@ -0,0 +1,39 @@
+# getting_started
+
```

Comment:
Could we add a table of contents- a list of items which link to their entry in the doc. I know we only have 1 item at the moment, but will likely add more